code/__DEFINES/mobs.dm

```diff
@@ -438,6 +438,7 @@ var/list/global_mutations = list() // list of hidden mutation things
 #define BASE_HALLOSS_RECOVERY_RATE -2
 #define REST_HALLOSS_RECOVERY_RATE -10
 
+
```

Comment:
Try to clean your code, not leaving unintended modifications.

Reply:
?

Reply:
Check the changes. Your PR is adding a whitespace in this file. It's clearly something you added and then removed, leaving a space. Try to clean after your changes.